srcpkgs/keepassxc/template

```diff
@@ -1,45 +1,28 @@
 # Template file for 'keepassxc'
 pkgname=keepassxc
-version=2.3.4
+version=2.4.0
 revision=1
 build_style=cmake
-configure_args="-DWITH_TESTS=OFF
- -DWITH_XC_AUTOTYPE=$(vopt_if autotype ON OFF)
- -DWITH_XC_NETWORKING=$(vopt_if network ON OFF)
- -DWITH_XC_YUBIKEY=$(vopt_if yubikey ON OFF)
- -DWITH_XC_BROWSER=$(vopt_if browser ON OFF)
- -DWITH_XC_SSHAGENT=$(vopt_if sshagent ON OFF)"
-makedepends="qt5-tools-devel libgcrypt-devel libargon2-devel
-	$(vopt_if autotype 'qt5-x11extras-devel libXtst-devel libXi-devel')
-	$(vopt_if browser libsodium-devel)
-	$(vopt_if network libcurl-devel)
-	$(vopt_if yubikey 'libykpers-devel libyubikey-devel')"
+configure_args="-DWITH_TESTS=OFF -DWITH_XC_ALL=ON"
+makedepends="qt5-tools-devel qt5-svg-devel libgcrypt-devel libargon2-devel qrencode-devel
+	qt5-x11extras-devel libXtst-devel libykpers-devel libyubikey-devel libsodium-devel"
 short_desc="KeePassXC is a cross-platform port of “Keepass Password Safe”"
 maintainer="ibrokemypie <ibrokemypie@bastardi.net>"
-license="GPL-3.0-only"
+license="GPL-3.0-or-later, BSD-3-Clause, CC0-1.0, LGPL-2.0-only, LGPL-2.1-only,
+	LGPL-3.0-or-later, Nokia-Qt-exception-1.1, MIT BSL-1.0"
```

Comment:
this indent is wrong now, inside functions is a tab, inside license= and makedepends= it is 1 space

Reply:
ok, I get it now